spec/models/email_address_spec.rb

```diff
@@ -1,5 +1,85 @@
 require 'rails_helper'
 
-describe EmailAddress do
+describe EmailAddress, type: :model do
   # format validation specs in `email_address_validations_spec.rb`
+  # email address provider validation specs in `domain_mx_validator_spec.rb`
+
+  INVALID_PROVIDER = "#{SecureRandom.hex(3)}.#{SecureRandom.hex(3)}"
+
+  let(:strategy) { double('validator') }
+
+  before(:each) do
+    EmailDomainMxValidator.strategy = strategy
+    # ignore missing translation errors
```

Comment:
why are there missing translations?

Reply:
Fixed. There were missing translation because I'm creating/adding a new kind of error: `missing_mx_records` and I hadn't added an entry in `config/locales/activerecord.en.yml`

Reply:
Can you (eventually, doesn't have to be this PR) ask folks at Katalyst to provide a translation for the polish locale file?

Reply:
Fixed. Copied the translation.